.ci/pitest.sh

```diff
@@ -215,6 +215,14 @@ pitest-utils)
   checkPitestReport "${ignoredItems[@]}"
   ;;
 
+pitest-xpath)
+  mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
+  declare -a ignoredItems=(
+  "ElementNode.java.html:<td class='covered'><pre><span  class='survived'>            if (parentDepth == -1) {</span></pre></td></tr>"
```

Comment:
We are not expecting new items as we are trying to shrink the list. Let me review this before giving more guidance.

Reply:
> removed conditional - replaced equality check with true → SURVIVED.

Just on the face, it looks like our tests don't see the benefit of the cache and goes back to directly recounting all the nodes. I don't see any way to make this matter under a normal condition. You will need to create a test specifically for this. Like make 2 nodes, have the parent node have a depth of 9 and count the depth of the child. Since there are only 2 nodes, it will have a count of 2 if recounted, but the actual count will be 10.

Reply:
Done, created a new UT.